addons/binding/org.openhab.binding.roomba/src/main/java/org/openhab/binding/roomba/model/exception/RoombaCommunicationException.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.roomba.model.exception;
+
+/**
+ * Represents an error communicating with the Roomba
+ * 
+ * @author Stephen Liang
+ *
+ */
+public class RoombaCommunicationException extends Exception {
```

Comment:
Do you need it a checked exception or could it be a RuntimeException, just personal preference. But a checked exception like this might `taint` the complete code.